application/common/sync/Synchronizer.php

```diff
@@ -105,6 +105,12 @@ public function syncAll()
         }
     }
     
+    /**
+     * Synchronize a specific user, requesting their information from the
+     * ID Store and updating it accordingly in the ID Broker.
+     *
+     * @param string $employeeId The EmployeeID of the user to sync.
+     */
     public function syncUser($employeeId)
     {
         $idStoreUser = $this->idStore->getActiveUser($employeeId);
```

Comment:
Is there an expectation with the idStore component that each use record is of a specific interface/type? On line 87 it is hard coded to expect an array property of `employee_id`, but what if it was working with a different ID Store, would it still have an `employee_id` property or will this break us later?